fe/src/App.tsx

```diff
@@ -15,6 +16,7 @@ function App(): JSX.Element {
             <Route exact path="/" component={Home} />
             <Route path="/about" component={About} />
             <Route path="/Login" component={Login} />
+            <Route path="/ProductPage" component={ProductPage} />
```

Comment:
Path to product creation should not be called "ProductPage".
Something like `/product/new` or `/product/create` makes more sense.

Also consider renaming of `ProductPage` component itself. If it fills the purpose of "Creating a product", then perhaps `ProductCreate` or something similar suits it more.